sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/impl/RestServiceStarter.java

```diff
@@ -59,10 +58,10 @@ public void start() throws Exception {
     // pass whatever we want/need
 
     // Then Metrics-related things: hard-coded initially
-    final Metrics metricsImpl = new MetricsImpl();
+    final MetricsImpl metricsImpl = new MetricsImpl();
```

Comment:
So to enable percentiles you should call the `MeterRegistryConfiguration.configure(meterRegistry);` using the `meterRegistry` from the `MetricsImpl`..

However, until now this was done in the constructor by calling the `MetricsImpl#initPrometheusMeterRegistry` to configure everything.. But I guess the `MetricRegistry` is out of the play now, so you maybe changed some code in the `MetricsImpl` for the v2?

Anyway this is solvable easily.

Reply:
I use `MetricsImpl` exactly as-is with no changes so far. But initialization/construction may have changed.

Odd thing is that percentiles seem to be added for some metrics by default, but not for specific ones being tested.
At least when running thing locally and having a look at `/metrics` endpoint.
So I guess I do not understand logic of them getting added or not.

Reply:
I think `MetricsRegistry` should also exist.

But one thing I was wondering about was the whole way in which static initialization was called from non-static context. And whether that ordering might be fragile so that seemingly unrelated change in set up of other components might trigger changes in timing of metrics registry initialization.

Reply:
I am not sure about those changes; @jeffreyscarpenter is bit more familiar with it, but I think this was straight copy-paste to split REST metrics test from others.

Reply:
I am missing the context. What code was copy-pasted?

Reply:
@jeffreyscarpenter  There was a test class like `MetricsTest`, containing tests for metrics of all APIs/services; a new one (like `RestApiMetricsTest`) was extracted from that. I was thinking you had done the initial part earlier and might have context.

Reply:
Yes the copy paste is fine, what about support for the `@StargateSpec(parametersCustomizer = "buildParameters")`? So I was not lazy and debugged this:

* The second JVM that you fire in the INT tests and that I guess represents the `REST v2` is completely ignoring the those properties
* It's because `RestApiExtension` has different way of supplying properties and this was not included in the test

Anyway I fixed this in #1571